LICENSE

```diff
@@ -1,201 +1,15 @@
-                                 Apache License
-                           Version 2.0, January 2004
-                        http://www.apache.org/licenses/
+Copyright 2019 Lightbend, Inc.
 
-   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
+This software is licensed under the Apache 2 license, quoted below.
 
-   1. Definitions.
+Licensed under the Apache License, Version 2.0 (the "License"); you may not
+use this file except in compliance with the License. You may obtain a copy of
+the License at
 
-      "License" shall mean the terms and conditions for use, reproduction,
-      and distribution as defined by Sections 1 through 9 of this document.
+    [http://www.apache.org/licenses/LICENSE-2.0]
```

Comment:
Is there any concern that that link may not work at some point, thus making the LICENSE unreadable in full?

Reply:
Good question.  Actually, yes this looks like the better example to follow:
https://github.com/lightbend/console-charts/blob/master/LICENSE
The Copyright is at the end.
I had seen multiple projects using https://github.com/lightbend/cinnamon-ui/blob/master/LICENSE, but the full license is better. Thanks.